### ISSUE TITLE
Integrating this directly in to A4 workbench?

### ISSUE THREAD

**Issue by luzpaz** [Ediforce44/FreeCAD-Assembly4-BOM#2]

**Comment by luzpaz** [comment]
bump

**Comment by Ediforce44** [comment]
Hey I don't know how the forum works. Is this just a email notification
because I couldn't find a post named like that. Or is this a question and
you want a anwser from me :D

luzpaz ***@***.***> schrieb am Di., 11. Okt. 2022, 15:19:

**Comment by Ediforce44** [comment]
Yes, this would be a good idea. But my version is very specific I would say. Should I contact someone?

**Comment by Ediforce44** [comment]
Maybe there are some changes needed for a good integration into A4. But i wouldn't refuse a direct integration into A4.

**Comment by luzpaz** [comment]
https://github.com/Zolko-123/ is the A4 dev

**Comment by Ediforce44** [comment]
Thanks I will contact him

**Comment by Zolko-123** [comment]
so, what's the situation with this ?

**Comment by julpec** [comment]
Yes… what's the situation ? This tool is so perfect (in theory… because it doesn't work with A4 v0.50.12)
